tooling/cli/src/interface/rust.rs

```diff
@@ -248,12 +251,12 @@ impl Interface for Rust {
     );
     env.insert(
       "TAURI_ENV_PLATFORM",
-      match host {
+      match (host, host_env) {
         // keeps compatibility with old `std::env::consts::OS` implementation
-        "darwin" => "macos".into(),
-        "ios-sim" => "ios".into(),
-        "androideabi" => "android".into(),
-        h => h.into(),
+        ("darwin", _) => "macos".into(),
+        ("ios", Some("sim")) => "ios".into(),
+        ("androideabi", _) => "android".into(),
```

Comment:
lets' remove these compatibilities and use them as is.

Reply:
did you mean like this? https://github.com/tauri-apps/tauri/pull/8448/commits/c3f12cd7851efa8b9b5b78f482691b47cff73aa4

Reply:
yeah, except maybe keep the conversion from `ios-sim` -> `ios`, what do you think @lucasfernog ?

Reply:
yeah, i left that one in. By splitting it into host and host_env this is done above already. I also didn't see much value in a seperate ios-sim target (unless androideabi is only used in sims?)

Reply:
Ah nice, didn't notice that. if anyone wants `-sim` part, they can parse `TAURI_ENV_TARGET_TRIPLE` themselves.